job-templates/kubernetes_release.json

```diff
@@ -3,7 +3,8 @@
   "properties": {
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
-      "orchestratorRelease": "1.13"
+      "orchestratorRelease": "1.14",
+      "azureCNIVersion": "v1.0.17"
```

Comment:
I don't see this documented at https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md, so checking now.

Reply:
It's not. It's defined in types but it doesn't really do what I assumed. Changing ASAP with azureCNIURLWindows / Linux

Reply:
yeah, it's overridden here https://github.com/Azure/aks-engine/blob/1d979960996dd2d07d5f9efd417bc0be8772a32b/pkg/api/defaults.go#L237 


URL works